games/u.yaml

```diff
@@ -560,6 +560,21 @@
   url: https://unvanquished.net
   feed: https://unvanquished.net/feed
 
+- name: Uplink
+  originals:
+  - Uplink
+  type: remake
+  repo: 'https://github.com/AdrUlb/Uplink'
+  development: sporadic
+  status: playable
```

Comment:
where does it show the game as playable? I don't see any significant code about game logic etc